src/effect_call_builder.ts

```diff
@@ -39,12 +38,10 @@ export class EffectCallBuilder extends CallBuilder<
    * @returns {EffectCallBuilder} this EffectCallBuilder instance
    */
   public forLedger(sequence: number | string): this {
-    this.filter.push([
+    return this.forEndpoint(
       "ledgers",
       typeof sequence === "number" ? sequence.toString() : sequence,
```

Comment:
nit: I think we can replace this expression with just `sequence.toString()`

Reply:
Will be fixed by #702 :+1: